bucket/krita.json

```diff
@@ -5,11 +5,11 @@
     "license": "GPL-3.0-or-later",
     "architecture": {
         "64bit": {
-            "url": "https://download.kde.org/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z",
+            "url": "http://mirror.kumi.systems/kde/ftp/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z",
```

Comment:
```suggestion
            "url": "http://download.kde.org/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z",
```

Reply:
Nothing was resolved. Use http for download.kde. instead of http from some third party vendor.

Reply:
I did try to do that before I made #2932, but unfortunately I get the same error.

```sh
Installing 'krita' (4.2.7.1) [64bit]
The remote server returned an error: (302) Found.
URL http://download.kde.org/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z is not valid
```

I used the same link as you suggested.

What do you suggest I do? Apologies for resolving the conversations so fast, I am a bit new to this.

Reply:
One of the [official KDE mirrors](https://download.kde.org/extra/download-mirrors.html) _could_ be used instead, based on your comment:

> Nothing was resolved. Use http for download.kde. instead of http from some third party vendor.

For example: https://mirrors.mit.edu/kde/stable/krita/4.2.7.1/krita-x64-4.2.7.1-setup.exe#/dl.7z

Unless I put a fixed URL inside [bucket/krita.json](https://github.com/lukesampson/scoop-extras/blob/master/bucket/krita.json), scoop fails to download and install Krita on the system.

And of course, I do actually test downloading Krita through multiple ways, of which download.kde failed to work for both **http** and **https** versions.